.github/ISSUE_TEMPLATE/config.yml

```
@@ -1,7 +1,7 @@
 blank_issues_enabled: true
 contact_links:
   - name: Trouble with your code? 😕
-    url: https://ibm.co/joinqiskitslack
+    url: http://qisk.it/join-slack
```

```suggestion
    url: https://qisk.it/join-slack
```

Should we use https, right?